docs/content_management/pages/page_block_attributes.md

```diff
@@ -176,9 +176,9 @@ ibexa_fieldtype_page:
                                 class: 'class1 class2'
 ```
 
-- `help` - defines a help message which is rendered below the field.
-- `help_attr` - sets the HTML attributes for the element which displays the help message.
-- `help_html` - set this option to `true` to disable escaping the contents of the `help` option when rendering in the template.
+- `help` - defines a help message which is rendered below the field
+- `help.attr` - sets the HTML attributes for the element which displays the help message
+- `help.html` - set this option to `true` to disable escaping the contents of the `help` option when rendering in the template
```

Comment:
```suggestion
- `help.text` - maps to [`help`](https://symfony.com/doc/current/reference/forms/types/form.html#help) and defines a help message which is rendered below the field
- `help.attr` - maps to [`help_attr`](https://symfony.com/doc/current/reference/forms/types/form.html#help-attr) and sets the HTML attributes for the element which displays the help message
- `help.html` - maps to [`help_html`](https://symfony.com/doc/current/reference/forms/types/form.html#help-html). Set this option to `true` to disable escaping the contents of the `help` option when rendering in the template
```

I think this part can be confusing, what do you think about mentioning this mapping expicitly?

If you agree with this, then the Symfony links should be replaced with a variable

Reply:
@mnocon I would swap the finality and the how to have the more interesting part first.

```suggestion
- `help.text` - defines a help message which is rendered below the field (maps to [`help`]([[= symfony_doc =]]/reference/forms/types/form.html#help))
- `help.attr` - sets the HTML attributes for the element which displays the help message (maps to [`help_attr`]([[= symfony_doc =]]/reference/forms/types/form.html#help-attr))
- `help.html` - enable (default) / disable (set to `true`) escaping the contents of the `help.text` option when rendering in the template (maps to [`help_html`]([[= symfony_doc =]]/reference/forms/types/form.html#help-html))
```